### PR TITLE
build: update plugin and dependencies

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -229,6 +234,36 @@
         <pluginManagement>
             <plugins>
                 <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
```
Comment (fanminshi) [inline pom.xml]: does maven include those clean, deploy, install by default?

Comment (lburgazzoli) [reply pom.xml]: yes. the reason I explicit set the version is that the default version is set in the root pom and may vary according to the maven version so it is a good idea to fix it in order to have the same behavior regardless of the maven version used.

Comment (fanminshi) [reply pom.xml]: sounds good!

**Comment by fanminshi** [comment]
lgtm

**Pull request merged**